.gitignore

```diff
@@ -13,8 +14,6 @@ libs
 # Classpath
 .classpath
 
-# Config properties
-config.properties
```

Comment:
Why are you no longer ignoring the config.properties? If it contains your auth token, it shouldn't be in VCS.